### PR TITLE
Preview column removed from cameras table

### PR DESCRIPTION
**Pull request by muneeb2542**
@azharmalik3

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -749,14 +749,14 @@ task :export_thumbnails_to_s3 do
...
-        puts "S3 export: Thumbnail for camera #{camera.exid} exported to S3"
-      end
+     # unless camera.preview.blank?
+     #   Evercam::Services.snapshot_bucket.objects.create(filepath, camera.preview)
+     #   image = Evercam::Services.snapshot_bucket.objects[filepath]
+     #   camera.thumbnail_url = image.url_for(:get, {expires: 10.years.from_now, secure: true}).to_s
+     #   camera.save
+     #
+     #   puts "S3 export: Thumbnail for camera #{camera.exid} exported to S3"
+     # end
```
Comment (mosic) [inline Rakefile]: If you want to remove code, always delete it, don't leave it commented out.

**Comment by mosic** [comment]
@azharmalik3 @muneeb2542 Heads up, some tests are failing on Travis.

**Comment by muneeb2542** [comment]
@mosic  yes we know about this i am currently working on it.

**Comment by mosic** [comment]
@muneeb2542 The Rake task `export_thumbnails_to_s3` you modified is useless if that block is removed, it's better to delete the whole task in order to avoid confusion.

**Comment by muneeb2542** [comment]
@mosic i have removed the export_thumbnails_to_s3 task altogether.

**Pull request merged**